steps/aws-secrets-manager/1.0.0/step.yml

```diff
@@ -0,0 +1,62 @@
+title: Fetch secrets
+summary: Fetch secrets from AWS Secrets Manager
+description: |
+  The main intention of this custom step is to allow storage of sensitive data on AWS Secrets Manager, and fetching them on-demand during Bitrise workflow execution.
```

Comment:
NOTE: step should start with a capital letter: `Step` and I would drop the `custom` word, as it is a 'regular' Bitrise Step.

Reply:
Addressed in moneylion/bitrise-step-aws-secrets-manager@b843ca6bebd22d36aef62bd358ee0ece2dd01901.